packages/components/src/filters/advanced/index.js

```diff
@@ -178,6 +179,15 @@ class AdvancedFilters extends Component {
 										query={ query }
 									/>
 								) }
+								{ 'Number' === input.component && (
+									<NumberFilter
+										filter={ filter }
+										config={ config.filters[ key ] }
+										onFilterChange={ this.onFilterChange }
+										isEnglish={ isEnglish }
```

Comment:
Not specific to this PR, but I'm curious what this does.  Looks like it sets specific grid widths for English.  Is the plan to set column widths for all languages or is there something specific to English that we need to do this for? /cc @psealock

Reply:
@jeffstieler @joshuatf Good question.

It has to do with column widths. The designs include a specified width for columns 1, 2 and 4 (the X to remove filter). Column 3, in the case below takes up the remaining space:

<img width="792" alt="screen shot 2018-09-25 at 4 29 02 pm" src="https://user-images.githubusercontent.com/1922453/45993243-0b540e00-c0e2-11e8-9674-0b634f3cf01c.png">

When dealing with non-english sentences, order can't be guaranteed. So when `isEnglish === false`, the layout changes such that Columns 1 and 2 have no defined width:

<img width="794" alt="screen shot 2018-09-25 at 4 27 28 pm" src="https://user-images.githubusercontent.com/1922453/45993248-0ee79500-c0e2-11e8-8cc7-8f0cfd72bfcf.png">

More info here https://github.com/woocommerce/wc-admin/pull/457